sycl/test-e2e/AddressSanitizer/out-of-bounds/USM/unaligned_shadow_memory.cpp

```diff
@@ -0,0 +1,38 @@
+// REQUIRES: linux
```

Comment:
add granularity to test filename since this file is also testing it?

Reply:
Actually this test is just trying to increase the testing coverage of asan load/store algorithms.

Reply:
so is it right we are basically not able to test granularity in end-2-end test? Then the use of it in the test might be confusing.

Reply:
Yeah, since granularity is transparent to user, test it in e2e test is not easy. But the lit test added in PR https://github.com/intel/llvm/pull/13678/files#diff-3b53314d6162f2442b8470905daba8a7dfd9feedac33049431320249b313a4f1R8 can cover such granularity change.